DeepSpace/src/main/cpp/subsystems/Claw.cpp

```diff
@@ -0,0 +1,33 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2017-2018 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "subsystems/Claw.h"
+
+Claw::Claw() : Subsystem("Claw") {
+  m_clawSolenoid.reset(new frc::Solenoid(RobotMap::Claw::k_clawSolenoid));
+
+
+}
+
+void Claw::InitDefaultCommand() {
```

Comment:
Do you plan on using this function, if not I think we should delete it from the .cpp and the .h files

Reply:
FIRST wants us to have that function, and suggests that we don't use it if we don't want it. I'm ok with keeping it in there in case something is calling that on each subsystem. source: https://wpilib.screenstepslive.com/s/3120/m/7952/l/129364-default-commands

Reply:
Are we all merged into master?

Reply:
I believe we have decided to not push code that has not been tested on the robot. There is no way to test if the code works as there are no solenoids installed on the robot, nor is the compressor wired up.